R/exploratoryfactoranalysis.R

```diff
@@ -418,6 +509,72 @@ ExploratoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
 
 }
 
+# Modification here:
+# Creates a table that outputs the details of the parallel analysis.
+# Particularly useful for reporting the rationale of how many factors were
+# retained - especially within psychometric papers.
+# We are also going to be adding an asterisk aside any factor whose real data eigenvalue is above
+# the 95th percentile simulated data eigenvalue
+.efaPATable <- function(modelContainer, dataset, options, ready) {
+  if (!options[["incl_PAtable"]] || !is.null(modelContainer[["patab"]])) return()
+
+  if (options[["basedOn"]] == "mixed") {
+    polytetracor <- psych::mixedCor(dataset)
+    pa <- try(psych::fa.parallel(polytetracor$rho,
+                                 plot = FALSE,
+                                 fa = options$parallelMethod,
```

Comment:
```suggestion
                                 fa = options$parallelMethodTable,
```
Based on the changes in the qml file